hazelcast/src/test/java/com/hazelcast/cache/CacheListenerTest.java

```diff
@@ -51,7 +50,7 @@
 import static org.junit.Assert.fail;
 
 @RunWith(HazelcastSerialClassRunner.class)
-@Category({QuickTest.class, ParallelJVMTest.class})
```

Comment:
Why did you remove this? The test does not extend another test class.

Reply:
But this class is extended - by `CacheClientListenerTest`, and that class can't be `ParallelJVMTest` because it uses real client.